src/services/ffmpegService.ts

```diff
@@ -56,6 +56,38 @@ class FFmpegService {
             }
         }
     }
+
+    async convertLivePhotoToMP4(file: Uint8Array): Promise<Uint8Array> {
+        if (!this.ffmpeg) {
```

Comment:
FFmpeg can execute only one command at a time if the user switches between two live photos this function will be called for each of them and will throw an error.

use the queue processor to queue up the task and wait on the returned promise